web-api-shared/src/elastic.rs

```diff
@@ -367,10 +527,24 @@ pub enum Error {
         url: Url,
         error: String,
     },
-    /// Failed to serialize a requests or deserialize a response.
+    /// Failed to serialize a requests or deserialize a response: {0}
     Serialization(serde_json::Error),
     /// Given endpoint was not found: {0}
```

Comment:
:art: we should update the error message as well
```suggestion
    /// Given resource was not found: {0}
```